src/index.js

```diff
@@ -148,10 +149,17 @@ if ( !! window.wcCalypsoBridge.isEcommercePlanTrial ) {
 		render: WoocommercePaymentsTaskPage,
 	} );
 
-	registerPlugin( 'wc-calypso-bridge-homescreen-slotfill-banner', {
-		render: CalypsoBridgeHomescreenBanner,
-		scope: 'woocommerce-admin',
-	} );
+	if ( window.wcCalypsoBridge.introductoryOffer ) {
```

Comment:
Since this particular component is specific for ecom free trial plan, I think it's good to add ecom trial check here as well.

Reply:
I believe the check is already done here https://github.com/Automattic/wc-calypso-bridge/blob/78bf2dcc2cb3f625428cf8315ae24e1dfeeb2472/src/index.js#L102